README.md

```diff
@@ -4,16 +4,52 @@
 
 The PostgreSQL Operator deploys and operates the [PostgreSQL](https://www.postgresql.org/about/) database on machine clusters.
 
-This operator provides a Postgres database with replication enabled (one master instance and one or more hot standby replicas). The Operator in this repository is a Python script which wraps Postgres versions shipped by the Ubuntu bionic and focal series, providing lifecycle management and handling events (install, configure, integrate, remove).
+This operator provides a Postgres database with replication enabled (one master instance and one or more hot standby replicas). The Operator in this repository is a Python script which wraps Postgres versions shipped by the Ubuntu focal series, providing lifecycle management and handling events (install, configure, integrate, remove).
 
 ## Usage
 
-As this charm is not yet published, you need to follow the build and deploy instructions from [CONTRIBUTING.md](CONTRIBUTING.md).
+To deploy this charm using Juju 2.9.0 or later, run:
 
-## Accessing the database
+```shell
+juju add-model postgresql
+charmcraft pack
+juju deploy ./postgresql_ubuntu-20.04-amd64.charm
+```
 
-You can access the database using any PostgreSQL client by connecting on the unit address and port `5432` as user `postgres` with the password shown by the command below.
+Note: the above model must exist outside of a k8s environment (you could bootstrap an lxd environment).
 
-```bash
-juju run-action postgresql/0 get-initial-password --wait
-```
\ No newline at end of file
+To confirm the deployment, you can run:
+
+```shell
+juju status --color
+```
+
+Once PostgreSQL starts up, it will be running on the default port (5432).
+
+If required, you can remove the deployment completely by running:
+
+```shell
+juju destroy-model -y postgresql --destroy-storage
+```
+
+Note: the `--destroy-storage` will delete any data persisted by PostgreSQL.
+
+## Relations
+
+This charm implements the [provides data platform library](https://charmhub.io/data-platform-libs/libraries/database_provides), with the `mysql_client` interface.
+To relate to it, use the [requires data-platform library](https://charmhub.io/data-platform-libs/libraries/database_requires).
+
+Adding a relation is accomplished with:
+
+```shell
+# Deploy the relevant charms.
+charmcraft pack
+juju deploy ./postgresql_ubuntu-20.04-amd64.charm
+juju deploy mycharm
+# Relate mycharm with postgresql.
+juju relate mycharm:database postgresql:database
+```
```

Comment:
charm was deployed in previous steps
```suggestion
```shell
juju relate mycharm:database postgresql:database
```
```

Reply:
Fixed in https://github.com/canonical/postgresql-operator/pull/15/commits/20aeab67061d337050c07ce9d13a939c6fc405db.